docs/dg/dev/architecture/architectural-convention.md

```diff
@@ -1532,6 +1532,7 @@ The schema file defines the module's tables and columns. Schema files are organi
   - The field can be `null` by default.
   - The field needs to be eventually unique across the table; until a unique value is provided, the business logic may not operate appropriately.
 - Table foreign key definitions needs to include the `phpName` attribute.
+- Each field **must** have a description of the fields value; This helps understand the data model and the internal usage of a fields value. F.e. name="price" description="The base unit of a currency f.e. for the currency EUR it is cent."`.
```

Comment:
So the `convention` section indeed uses the `must` keywords, because it means that if you are not following it, you break something. However these changes are nice to haves (nothing gets broken if things are not documented :(  ), you need to move this to the guidelines with `should`.

Reply:
As discussed, breaking User Experience has to be taken into account as well. So we will stick with `must` here for core development. 

The Convention directive description states `must adhere to enable specific Spryker features` thus I think I don't need to change the text here.

You said that it's written that conventions automatically have to be seen as guidelines for projects. Where is that?